sklearn/cluster/k_means_.py

```diff
@@ -222,11 +222,10 @@ def k_means(X, n_clusters, init='k-means++', precompute_distances='auto',
     tol : float, optional
         The relative increment in the results before declaring convergence.
 
-    random_state : int, RandomState instance or None, optional, default: None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance or None (default)
+        Determines random number generation for centroid initialization. It
+        also generates random seeds for the parallel execution.
```

Comment:
I'm not sure this means anything for the user, if those seeds are then used only for init